HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRWirebug.cs

```diff
@@ -14,6 +14,7 @@ public sealed class MHRWirebug : IEventDispatcher, IUpdatable<MHRWirebugExtrasSt
     private double _maxTimer;
     private double _cooldown;
     private double _maxCooldown;
+    private double _maxExtraCooldown = 0.0;
```

Comment:
Unnecessary field as it's only being used inside the `Update` method, that can be a local variable.
```suggestion

```